setupext.py

```diff
@@ -455,6 +455,7 @@ def get_package_data(self):
                 *_pkg_data_helper('matplotlib', 'tests/tinypages'),
                 'tests/cmr10.pfb',
                 'tests/mpltest.ttf',
+                'tests/FreeSerif.ttf',
```

Comment:
the file is quite large; can you just subset the relevant glyphs eg. using fontforge as you did in #905?

Reply:
I did that as an amend commit and a force push, so that we don't get the full font in the history. In my first attempt I forgot some of the constituent glyphs, but now this should be as good a test as the full font.

Reply:
thanks :)